webrtc-stats.html

```diff
@@ -1528,6 +1561,22 @@ <h2>
                   corresponding {{RTCAudioPlayoutStats}}.
                 </p>
               </dd>
+              <dt>
+                <dfn>powerEfficientDecoder</dfn> of type <span class=
+                "idlMemberType">boolean</span>
+              </dt>
+              <dd>
+                <p class="fingerprint">
+                  Only defined when [= exposing hardware is allowed =].
+                </p>
+                <p>
+                  Whether the decoder currently used is considered power
+                  efficient by the user agent. This SHOULD reflect if the
+                  configuration results in hardware acceleration, but the user
+                  agent MAY take other information into account when deciding if
```

Comment:
Do we have examples of such information? Is it to cover the case of small resolutions for instance where some OSes may decide to go with software implementations

Reply:
It gives implementations some wiggle-room, like if a HW implementation isn't performant we could still report false without violating the spec. I don't know if this is a real problem in practise though.

I also think that it is OK for an implementation to say that SW is power efficient if the resolution is so small that SW would be more performant than HW, but I don't think finding that cutoff point is required of the implementation.